kettle/kettle.py

```diff
@@ -148,6 +154,8 @@ def refresh_choices(self):
 		self.queued_data.append(payload)
 
 	def refresh_options(self):
+		if isinstance(self.game.current_player, PlayerAI):
+			return
```

Comment:
You got that mixed up. Also, there's no reason not to send options - AI clients may want them.